README.md

```diff
@@ -17,6 +17,18 @@ Features
 
 ## Getting started
 
+### Nextcloud Code
+
+The nextcloud code base needs to be available including the `3rdparty` submodule.
+To clone it from github run:
+
+```
+git clone https://github.com/nextcloud/server.git
+cd server
+git submodule update --init
```

Comment:
To make sure we have it as easy as possible maybe add a comment about running `pwd` and and using the output in the further steps for the server path.